docs/decisions/0006-api-specification-for-enterprise-mfes.rst

```diff
@@ -0,0 +1,308 @@
+
+Sample API responses
+^^^^^^^^^^^^^^^^^^^^
+
+::
+
+  [
+    {
+      "course_run_key": "course-v1:ImperialX+dacc003+3T2019",
+      "redemption": {
+        "uuid": "26cdce7f-b13d-46fe-a395-06d8a50932e9",
+        "status": "fulfilled",
+        "policy_redemption_status_url": "/api/v1/policy/56744a36-93ac-4e6c-b998-a2a1899f2ae4/redemptions/26cdce7f-b13d-46fe-a395-06d8a50932e9/",
+        "courseware_url": "https://courses.edx.org/courses/course-v1:ImperialX+dacc003+3T2019/courseware/",
+        "errors": []
+      },
+      "subsidy_access_policy": {
+        "uuid": "56744a36-93ac-4e6c-b998-a2a1899f2ae4",
+        "policy_redemption_url": "/api/v1/policy/56744a36-93ac-4e6c-b998-a2a1899f2ae4/redeem/",
+        "policy_type": "LearnerCreditAccessPolicy",
+        "description": "Learner credit access policy",
+        "active": true,
+        "catalog_uuid": "14f701ea-7e0b-4a4e-bbda-f295e40c7bf1",
+        "subsidy_uuid": "7801b0ef-b1c2-4f3a-97fa-121f0bce48be",
+        "access_method": "direct",
+        "spent_limit": 10000,
+        "per_learner_spend_limit": 200,
+        "remaining_balance": 9500,
+        "remaining_balance_for_learner": 200
+      },
+      "errors": []
+    }
+  ]
+
+*No redeemable subsidy access policies available to the learner:*
+
+::
+
+  [
+    {
+      "course_run_key": "course-v1:ImperialX+dacc003+3T2019",
+      "redemption": null,
+      "subsidy_access_policy": null,
+      "errors": [
+        {
+          "code": 400,
+          "message": "Insufficient balance remaining",
+        }
+      ]
+    }
+  ]
+
+*Redeemable subsidy access policy that has not yet been redeemed and/or fulfilled:*
+
+::
+
+  [
+    {
+      "course_run_key": "course-v1:ImperialX+dacc003+3T2019",
+      "redemption": null,
+      "subsidy_access_policy": {
+        "uuid": "56744a36-93ac-4e6c-b998-a2a1899f2ae4",
+        "policy_redemption_url": "/api/v1/policy/56744a36-93ac-4e6c-b998-a2a1899f2ae4/redeem/",
+        "policy_type": "LearnerCreditAccessPolicy",
+        "description": "Learner credit access policy",
+        "active": true,
+        "catalog_uuid": "14f701ea-7e0b-4a4e-bbda-f295e40c7bf1",
+        "subsidy_uuid": "7801b0ef-b1c2-4f3a-97fa-121f0bce48be",
+        "access_method": "direct",
+        "spent_limit": 10000,
+        "per_learner_spend_limit": 200,
+        "remaining_balance": 9500,
+        "remaining_balance_for_learner": 200
+      },
+      "errors": []
+    }
+  ]
+
+*Redeemable subsidy access policy that has been redeemed but is pending fulfillment:*
+
+::
+
+  [
+    {
+      "course_run_key": "course-v1:ImperialX+dacc003+3T2019",
+      "redemption": {
+        "uuid": "26cdce7f-b13d-46fe-a395-06d8a50932e9",
+        "status": "pending",
+        "policy_redemption_status_url": "/api/v1/policy/56744a36-93ac-4e6c-b998-a2a1899f2ae4/redemptions/26cdce7f-b13d-46fe-a395-06d8a50932e9/",
+        "courseware_url": null,
+        "errors": []
```

Comment:
note: `errors` is the field name we want inside the `redemption` object - it can describe possible fulfillment failure modes.  This might actually be populated when the status is "pending".